app/views/search/_reverb_retirement_modal.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="modal fade sitetour" id="reverbRetirementModal" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <a href="#" class="close modal-close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></a>
+        <h2 class="modal-title" id="reverbRetirementModalLabel">Welcome to Earthdata Search</h2>
+      </div>
+      <div class="modal-body modal-body-welcome">
+        <p>You have been redirected to Earthdata Search because Reverb is being retired on January 1st, 2018. Earthdata Search is replacing Reverb. If you would like to go return to Reverb for the time being, you may do so by clicking "Return to Reverb" below. Or you can choose to use Earthdata Search for now. At any point you may opt to return back to Reverb. We will remind you periodically that Reverb is retiring as its retirement date draws closer. If you would like to take a tour of Earthdata Search's main features, you may do so after choosing to stay on Earthdata Search. If you would like a more in-depth demonstration of Earthdata Search, watch this recorded webinar.</p>
+      </div>
+      <div class="modal-footer">
+        <button class="button" data-bind="click: $root.ui.reverbRetirement.stayWithEDSC">Stay with EDSC <i class="fa fa-arrow-circle-left"></i></button>
+        <button class="button" data-bind="click: $root.ui.reverbRetirement.returnToReverb">Return to Reverb <i class="fa fa-arrow-circle-right"></i></button>
```

Comment:
Make these <a> instead of <button> and set ```href``` with the reverb url in config.

Reply:
An href is a direct link; at this step in the logic, the cookie has not been set.  The data-bind sends the logic to the 'hat' to do its work, which then forwards the user appropriately.

Reply:
A link set for an <a> tag is fine. Cookie should be set 'onclick', meaning it should happen only after user actually determines that he/she wants to stay/leave the site. (see 5) in the 'missing features' comment below)

Reply:
also we want to redirect the user to the reverb home page where it has a "convert the query to edsc" that user can use.

Reply:
@mightynimble Okay - now have the redirect to reverb to its home page instead of the full referrer URL